03-kogito-travel-agency/README.md

```diff
@@ -122,9 +123,11 @@ When using native image compilation, you will also need:
 
 #### Infinispan
 
-This application requires an Inifinispan server to be available and by default expects it to be on default port and localhost.
+This application requires an Infinispan server to be available and by default expects it to be on default port and localhost.
 
-You can install Inifinispan server by downloading it from [official website](https://infinispan.org/download) version to be used in 10.0.0.Beta4
+You can install Infinispan server by downloading it from [official website](https://infinispan.org/download) version to be used in 10.0.0.CR1
```

Comment:
@nmirasch I've been using CR2, could we update to that?

Reply:
updated!